Sources/Rendering/Core/ImageMapper/index.js

```diff
@@ -46,24 +48,24 @@ function vtkImageMapper(publicAPI, model) {
     image.worldToIndex(pos3, ijk);
 
     const ex = image.getExtent();
-    const { ijkMode } = publicAPI.getClosestIJKAxis();
     let slice = 0;
     switch (ijkMode) {
       case SlicingMode.I:
         slice = vtkMath.clampValue(ijk[0], ex[0], ex[1]);
-        slice = Math.round(slice);
         break;
       case SlicingMode.J:
         slice = vtkMath.clampValue(ijk[1], ex[2], ex[3]);
-        slice = Math.round(slice);
         break;
       case SlicingMode.K:
         slice = vtkMath.clampValue(ijk[2], ex[4], ex[5]);
-        slice = Math.round(slice);
         break;
       default:
         return 0;
     }
+
+    if (ijkMode === model.slicingMode) {
+      slice = Math.round(slice);
+    }
```

Comment:
Why redo the rounding here if the goal is to do continuous index?

Reply:
As discussed:
- this rounding only occurred if `slicingMode` was I, J or K
- however, this method should really only be called when in X, Y or Z (it does world to index given one single XYZ component, outputting one single IJK component). yes, the name of the method is terrible in hindsight
- either way, we'd want IJK to be able to be continuous also, and just fix the mapper to round the index before accessing the scalar data, so no rounding needed here anyway.